orc8r/tools/helm/package.sh

```diff
@@ -107,7 +107,7 @@ fi
 
 # Push charts
 git add . && git commit -m "orc8r charts commit for version $ORC8R_VERSION"
-git remote set-url origin $GITHUB_REPO_URL
+git config remote.origin.url >&- || git remote add origin $GITHUB_REPO_URL
```

Comment:
- What does this line do?
- As a general thing: if we're specifying a `bash` hashbang, we should be naming the file with the extension `.bash` rather than `.sh`

Reply:
It adds a remote origin if it isn't set. This ensures that the script will work for both first time installs and upgrades.